app/webpacker/css/admin_v3/components/select2.scss

```diff
@@ -182,7 +182,6 @@ label .select2-container {
 .select2-container {
   .select2-choice,
   .select2-choices {
-    height: $btn-relaxed-height !important; // !important is needed because of vendor/assets/stylesheets/select2.css.scss
```

Comment:
This rule was causing the choices to have a fixed height and hence the choices' height was not being adjusted as per the content in it.

Reply:
Great, I've always been against using `height` like this!
I recall that JB was using `height` to help line up elements in the new design, so it's possible that this will impact on the layout (eg on bulk products edit screen). I'll add a note for testers to check.